src/markdown.js

```diff
@@ -43,6 +43,42 @@ export default function (config, options = {}) {
 	if (options.footnotes !== false) {
 		md.use(markdownItFootnote, options.footnotes);
 
+		let allFootnotes = new Map(); // pageId → { label: footnote }
+
+		function extractFootnoteTexts (tokens, pageId) {
+			let footnotes = {};
```

Comment:
Won't this erase previous footnotes if `extractFootnoteTexts is called multiple times on the same page?